dbms/src/Storages/Page/V3/spacemap/SpaceMap.cpp

```diff
@@ -112,6 +112,11 @@ SpaceMap::SpaceMap(UInt64 start_, UInt64 end_, SpaceMapType type_)
     , end(end_)
     , log(&Poco::Logger::get("SpaceMap"))
 {
+    if (start != 0)
```

Comment:
Is there any code depends on this assumption now?

Reply:
1. `getupperboundary` need `start` be 0. 
2. make no sense if we create a `SpaceMap` with a start is not 0.